common/xfer/controls.go

```diff
@@ -18,10 +18,15 @@ type Request struct {
 
 // Response is the Probe -> App -> UI message type for the control RPCs.
 type Response struct {
-	Value  interface{} `json:"value,omitempty"`
-	Error  string      `json:"error,omitempty"`
-	Pipe   string      `json:"pipe,omitempty"`
-	RawTTY bool        `json:"raw_tty,omitempty"`
+	Value interface{} `json:"value,omitempty"`
+	Error string      `json:"error,omitempty"`
+
+	// Pipe specific fields
+	Pipe   string `json:"pipe,omitempty"`
+	RawTTY bool   `json:"raw_tty,omitempty"`
+
+	// Remove specific fields
+	RemovedNode string `json:"removedNode,omitempty"` // Set if node was removed
```

Comment:
The UI just needs to know that an action succeeded. Is that the case when there is no error? Then I think we can check the control type in the UI, and close the details panel if the control returns w/o error.